Tests/SonarScanner.MSBuild.Common.Test/SonarPropertiesTests.cs

```diff
@@ -32,41 +32,38 @@ public class SonarPropertiesTests
     /// Strings that are used to indicate arguments that contain non sensitive data.
     /// </summary>
     private static readonly IEnumerable<string> NonSensitivePropertyKeys =
-        SonarProperties.ScanAllWarningParameters.Concat(
-        [
-            SonarProperties.ClientCertPath,
-            SonarProperties.JavaExePath,
-            SonarProperties.SkipJreProvisioning,
-            SonarProperties.HostUrl,
-            SonarProperties.SonarcloudUrl,
-            SonarProperties.ApiBaseUrl,
-            SonarProperties.ConnectTimeout,
-            SonarProperties.SocketTimeout,
-            SonarProperties.ResponseTimeout,
-            SonarProperties.UserHome,
-            SonarProperties.LogLevel,
-            SonarProperties.Organization,
-            SonarProperties.OperatingSystem,
-            SonarProperties.Architecture,
-            SonarProperties.PluginCacheDirectory,
-            SonarProperties.ProjectBaseDir,
-            SonarProperties.ProjectBranch,
-            SonarProperties.ProjectKey,
-            SonarProperties.ProjectName,
-            SonarProperties.ProjectVersion,
-            SonarProperties.PullRequestBase,
-            SonarProperties.PullRequestCacheBasePath,
-            SonarProperties.SourceEncoding,
-            SonarProperties.Verbose,
-            SonarProperties.VsCoverageXmlReportsPaths,
-            SonarProperties.VsTestReportsPaths,
-            SonarProperties.WorkingDirectory,
-            SonarProperties.CacheBaseUrl,
-            SonarProperties.HttpTimeout,
-            SonarProperties.ScanAllAnalysis,
-            SonarProperties.SonarSources,
-            SonarProperties.SonarTests,
```

Comment:
There is nothing to change in this PR, but this cleanup makes it harder to notice that the sources and test properties were removed.

Reply:
I agree, but I had to revert the previous PR that disabled the analysis when `sources and `tests` are specified, as I wouldn't be able to test it.